DEHPEcosimPro/Views/Rows/MappingThing.xaml.cs

```diff
@@ -0,0 +1,42 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="MappingRow.xaml.cs" company="RHEA System S.A.">
```

Comment:
this does not correspond to the class name, also, can i suggest to make it `MappingThingControl` ?